docs/readme.md

```diff
@@ -1 +1,19 @@
-# Hello Formester
+---
+home: true
+heroText: Formester Docs
+tagline: To start using Formester, check out our Guide
+actionText: See our guide →
+actionLink: ./guide.md
```

Comment:
Somehow I am not able to find the actionText and actionLink on the home page

![image](https://github.com/formester/formester-docs/assets/69912046/03d595a2-a74c-4cb5-b1db-93aba02d6904)


https://deploy-preview-1--formester-docs.netlify.app/

Reply:
Yes action link is not working. I tried it many times its not showing the button